### DISCUSSION TITLE
Squert Event IDs cutted

### DISCUSSION THREAD

**Discussion by bit1290** [Security-Onion-Solutions/securityonion#5882] [Unsupported Versions] [answered]
Hello,

Can someone explain why are dashes above the event ids on squert wui when I do a search?

Thanks 
![Capture](https://user-images.githubusercontent.com/16745902/137296847-ac480ce4-5a68-491c-a798-fec0209a5482.JPG)
:)

**Comment by dougburks** [comment, marked as answer]
You are running an old version of Security Onion that is [past End Of Life and no longer supported](https://docs.securityonion.net/en/2.3/eol.html). Please migrate to the new Security Onion 2. You can do a [fresh installation](https://docs.securityonion.net/en/2.3/getting-started.html) (recommended) or try an [in-place upgrade](https://docs.securityonion.net/en/2.3/appendix.html).
